.github/workflows/release-admin-tools.yml

```diff
@@ -46,15 +47,32 @@ jobs:
           go-version-file: "src/go.mod"
       - name: Calculate admintools tag
         run: |
+          get_tag() {
+            # We need to remove the `shallow` marker for this submodule or else
+            # `git describe --tags` won't find our tags
+            cd "target/$1"
+            # We need to fetch the tags before git describe will do what we want
+            # We _only_ fetch the tags to save time
+            git fetch origin 'refs/tags/*:refs/tags/*' >&2
+            rm -f ../.git/modules/$1/shallow
```

Comment:
we don't want to remove the `shallow` before the fetch? I'm not sure what it does exactly, just guessing